osquery/tables/system/linux/usb_devices.cpp

```diff
@@ -59,6 +60,29 @@ QueryData genUSBDevices(QueryContext &context) {
     r["vendor_id"] = UdevEventPublisher::getValue(device, kUSBKeyVendorID);
     r["serial"] = UdevEventPublisher::getValue(device, kUSBKeySerial);
 
+    // This will be of the form class/subclass/protocol and has to be parsed
+    auto devType = UdevEventPublisher::getValue(device, kUSBKeyType);
+    size_t pos = 0;
+    size_t last = 0;
+    std::vector<std::string> classInfo;
+    while (last < devType.size() &&
```

Comment:
Wait, instead of all of this:

At the top:
```cpp
#include "osquery/core/conversions.h"
```

Then:
```cpp
auto classInfo = osquery::split(devType, "/");
if (classInfo.size() == 3) {
  ...
```
```